auth/src/main/java/org/aerogear/mobile/auth/authenticator/AuthenticateOptions.java

```diff
@@ -0,0 +1,7 @@
+package org.aerogear.mobile.auth.authenticator;
+
+/**
+ * Represents options for an authentication action.
+ */
+public interface AuthenticateOptions {
```

Comment:
If this is only for Auth Response handling, maybe we could mention that in the class name? AuthenticateOptions sounds like something used to configure the auth request to login, rather than handle the auth response. Might just cause some confusion.

Reply:
It is not just for handing the auth response though. This is basically a wrapper to allow developer to provider their own configuration for certain things. For example, we could extend the implementation class to allow override some AppAuth configurations.

Reply:
Cool, that makes better sense.